_posts/2020-03-02-tomcat-thread-model.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Brakuje mi (oczywiście o ile jest to możliwe), linku do aplikacji testowej, bo czytając jakieś artykuły często szukam linku do kodu źródłowego.

Reply:
Ta testowana aplikacja to było Eximee, nie chciałbym jednak ujawniać kodów :-), Aplikacja testująca to taki najprostszy skrypt dla gatlinga - nie ma się czym chwalić.